app/react/Markdown/components/ContactForm.js

```diff
@@ -22,12 +21,16 @@ export class ContactForm extends Component {
     this.setState(changedData);
   }
 
-  submit(e) {
+  async submit(e) {
     e.preventDefault();
-    api.post('contact', this.state).then(() => {
+    try {
+      await api.post('contact', new RequestParams(this.state));
       this.props.notify('Message sent', 'success');
       this.setState({ name: '', email: '', message: '' });
-    });
+    } catch (err) {
+      return err;
+    }
+    return true;
```

Comment:
Not sure what returning the err does on a submit, or the convention to return `true`.  Is this a usual HTML convention?

At any rate, neither the success return nor the error cases are tested.

Reply:
There was no need for these returns. I simplified them and let the middlewares handle the errors.